connectivity/tests/clustermesh-endpointslice-sync.go

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package tests
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cilium/cilium-cli/connectivity/check"
+	"github.com/cilium/cilium-cli/utils/features"
+)
+
+func ClusterMeshEndpointSliceSync() check.Scenario {
+	return &clusterMeshEndpointSliceSync{}
+}
+
+type clusterMeshEndpointSliceSync struct{}
+
+func (s *clusterMeshEndpointSliceSync) Name() string {
+	return "clustermesh-endpointslice-sync"
+}
+
+func (s *clusterMeshEndpointSliceSync) Run(ctx context.Context, t *check.Test) {
+	ct := t.Context()
+	client := ct.RandomClientPod()
+
+	service, ok := ct.EchoServicesAll()[check.EchoOtherNodeDeploymentHeadlessServiceName]
+	if !ok {
+		t.Fatal("Cannot get echo-other-node-headless service")
```

Comment:
The const could be used in this error message, rather than a string literal with the name of the service.

Reply:
Indeed, thanks :D